bitcoin/src/blockdata/opcodes.rs

```diff
@@ -411,6 +411,22 @@ impl All {
     /// Encodes [`All`] as a byte.
     #[inline]
     pub const fn to_u8(self) -> u8 { self.code }
+
+    /// Encodes PUSHNUM [`All`] as a u8 representing its
+    /// number (1-16).
+    ///
+    /// Returns `None` if it is not a PUSHNUM.
+    /// It does not convert `OP_FALSE` to 0.
+    /// Only `OP_PUSHNUM_1` to `OP_PUSHNUM_16` are covered.
```

Comment:
```suggestion
    /// Encodes PUSHNUM [`All`] as a `u8` representing its number (1-16).
    ///
    /// Does not convert `OP_FALSE` to 0. Only `1` to `OP_PUSHNUM_16` are covered.
    ///
    /// # Returns
    ///
    /// Returns `None` if `self` is not a PUSHNUM.
    ///
    /// # Examples
    ///
    /// ```
    /// use bitcoin::opcodes::all::*;
    /// assert_eq!(OP_PUSHNUM_5.decode_pushnum().expect("pushnum"), 5)
    /// ```
```